### DISCUSSION TITLE
Any chance this comes to Firefox?

### DISCUSSION THREAD

**Discussion by Richard-Quinlivan** [Anarios/return-youtube-dislike#361] [Q&A] [answered]
I would love to use this extension, But I don't use a chromium based browser, I use Firefox, is there any chance that this extension will be available on Firefox? And if so, do you have any idea of the timetable on that?
If not I totally get it, just wondering if anything is in the works for that or if you all are focusing on other things. And even if it's not in the works, if you'd consider adding it to Firefox. I have no idea of the work load involved in doing that, but if it's not too much trouble, that would be awesome.

**Comment by himanshudabas** [comment, marked as answer]
https://www.returnyoutubedislike.com/install

**Reply by Richard-Quinlivan** [reply]
I swear I couldn't find it on the Firefox extension store before, but I see it now. Thanks
